### DISCUSSION TITLE
EA and ES Cluster

### DISCUSSION THREAD

**Discussion by buratinopy** [jertel/elastalert2#523] [Ideas]
Hello!
The EA code is hardcoded to connect to only one ES node
 elastalert/__init__.py [19]: `host=conf['es_host']`
config: `"es_host": "es_standalone_host.corp"`

We need to be able to connect to the ES cluster with the following config:
```
"es_hosts": [
   "es_cl0_n0.corp",
   "es_cl0_n1.corp",
   "es_cl0_n2.corp",
]
```
For compatibility in the operation of both methods, it is enough to change 2 lines in the file elastalert/init.py:
```
        super(ElasticSearchClient, self).__init__(host=conf.get('es_host', None),
                                                  hosts=conf.get('es_hosts', []),
```

https://github.com/jertel/elastalert2/commit/246aa141e5c55a4834e69eb5f185f10a30844341

**Comment by buratinopy** [comment]
ready to make a pull request

**Reply by jertel** [reply]
Your PR is welcomed, please ensure all [contribution guidelines](https://github.com/jertel/elastalert2/blob/master/CONTRIBUTING.md) are followed for the quickest merge.

Also, for background context, please review https://github.com/Yelp/elastalert/pull/3109.
